wisp-feature-testing/README.md

```diff
@@ -0,0 +1,13 @@
+# wisp-feature-testing
+
+Fake implementation of FeatureFlags.
+
+See [wisp-feature](https://github.com/cashapp/misk/tree/master/wisp-feature) for more details on
+feature flags.
+
+## Usage
+
+```kotlin
+
+// TODO - usages...
```

Comment:
Would be good to mention that it's for test purposes, and to have a usage doc here to validate that the API is reasonable.